modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/Conditions.tsx

```diff
@@ -189,6 +189,7 @@ export function Conditions({
 								'no-fields-were-found'
 							)}
 							error={errors.outputType}
+							id="conditions"
```

Comment:
Hi @Artartur. Please, rename the id to `objectValidationConditions`.

Reply:
Done!